package.json

```diff
@@ -81,7 +81,8 @@
     "tabWidth": 2
   },
   "lint-staged": {
-    "**/*.{js,jsx,ts,tsx}": [
+    "**/*.{js,jsx,json,ts,tsx}": [
```

Comment:
Just to double check - prettier respects the `.editorconfig` file with regards to `.json` formatting?

Reply:
Sadly it does not 😞 

Anyway, I don't think this is a good solution. `microbundle` keeps modifying the mangle.json and it's gonna be an issue keep them in sync. I think it'll be best to just waited out on `microbundle@0.12.0`. What do you think?

Reply:
Dang :/ Yea, perhaps we should wait.